src/core_plugins/kibana/public/management/sections/indices/index_pattern_list/components/list/list.js

```diff
@@ -51,6 +53,29 @@ export class List extends Component {
           })
         }
       </div>
+    ) : null;
+  }
+
+  renderNoDefaultMessage() {
```

Comment:
What do I need to do to get this to display? I tried deleting the default index pattern but then it just reassigned another one as default.

Reply:
have no index patterns defined